recipes-multimedia/libcamera-apps/rpi-libcamera-apps_git.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "A suite of libcamera-based apps for the Raspberry Pi"
+DESCRIPTION = "This is a small suite of libcamera-based apps that aim to \
+copy the functionality of the existing \"raspicam\" apps."
+HOMEPAGE = "https://github.com/raspberrypi/libcamera-apps"
+SECTION = "console/utils"
+
+SRC_URI = "\
+    git://github.com/raspberrypi/libcamera-apps.git;protocol=https;branch=main \
+"
+SRCREV = "22a52590c33a813743b4e6337478c208201c77b1"
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://license.txt;md5=a0013d1b383d72ba4bdc5b750e7d1d77"
+
+S = "${WORKDIR}/git"
+
+COMPATIBLE_MACHINE = "^rpi$"
+
+DEPENDS = "libcamera libexif jpeg tiff libpng boost"
+RDEPENDS:${PN} = "python3-core libcamera libexif jpeg tiff libpng"
+
+inherit cmake pkgconfig
+
+EXTRA_OECMAKE = "\
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBoost_INCLUDE_DIR=${STAGING_INCDIR} \
+    -DCMAKE_LIBRARY_PATH=${STAGING_LIBDIR} \
+    ${@bb.utils.contains("DISTRO_FEATURES", "x11", "-DENABLE_X11=1", "-DENABLE_X11=0", d)} \
+    ${@bb.utils.contains("MACHINE", "raspberrypi3", "-DENABLE_COMPILE_FLAGS_FOR_TARGET=armv8-neon", "", d)} \
+    ${@bb.utils.contains("MACHINE", "raspberrypi4", "-DENABLE_COMPILE_FLAGS_FOR_TARGET=armv8-neon", "", d)} \
+"
+
+PACKAGECONFIG[qt] = "-DENABLE_QT=1"
+PACKAGECONFIG[opencv] = "-DENABLE_OPENCV=1"
+PACKAGECONFIG[tensorflow-lite] = "-DENABLE_TFLITE=1"
+
+FILES:${PN} = "${bindir} \
```

Comment:
Are these needed?

Reply:
Are you referring to PACKAGECONFIG for opencv and tensorflow lite?

Reply:
No, I'm referring to `FILES:${PN}` definition. Wouldn't bindir be part of the defaults?

Reply:
Not sure - will test without it.

Reply:
Get the following build error without bindir

```log
ERROR: rpi-libcamera-apps-git-r0 do_package: QA Issue: rpi-libcamera-apps: Files/directories were installed but not shipped in any package:
  /usr/bin/libcamera-raw
  /usr/bin/camera-bug-report
  /usr/bin/libcamera-vid
  /usr/bin/libcamera-jpeg
  /usr/bin/libcamera-still
  /usr/bin/libcamera-hello
```

Reply:
@tewarid This doesn't seem right. First of all, _bindir_ is already there: https://git.yoctoproject.org/poky/tree/meta/conf/bitbake.conf#n327 . So those should be in by default - did you by any chance override it only with _libdir_? That would explain the error above. Same for _libdir_. It should work from defaults using SOLIBS (see the same link above).

Reply:
Resolved it it was dropped from the current version.